src/utilities/util_printlog.c

```diff
@@ -24,21 +24,47 @@ util_printlog(WT_SESSION *session, int argc, char *argv[])
     WT_DECL_RET;
     uint32_t flags;
     int ch;
+    // int start_lsn = 0;
+    WT_LSN start_lsn;
```

Comment:
All of the duplicate variable types can be moved onto one line, however its important to order them alphabetically to be consistent with the style in the codebase. Additionally the declaration and initial assignment should be on different lines.

So something like this:
```
    WT_LSN end_lsn, start_lsn;
    uint32_t  end_lsnfile,  end_lsnoffset, start_lsnfile, start_lsnoffset;
    int n_args;
    bool end_set, start_set;
    char *ofile, *start_str;

    end_set = start_set = false;
```